scripts/docs/metrics-to-integrations-repo

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+from pathlib import Path
+import string
+import yaml
+
+SCRIPT_DIR = Path(__file__).parent.resolve()
+AGENT_ROOT = SCRIPT_DIR / "../.."
+
+INTEGRATIONS_REPO = os.environ.get("INTEGRATIONS_REPO", (AGENT_ROOT / "../integrations").resolve())
+
+# Key is of the form <monitor/package/path:monitorType>, value is the
+# integration directory in the integrations repo where metrics should go (not
+# including the `docs/` path suffix).
+MONITORS_TO_INTEGRATIONS_DIR = {"internal/monitors/elasticsearch:elasticsearch": "elasticsearch"}
+
+METRIC_MARKDOWN_TEMPLATE = string.Template(
+    """
+---
+title: $title
+brief: $brief
+metric_type: $metric_type
+custom: $custom
+---
+### $title
+
+$description
+"""
+)
+
+
+def do_sync():
+    for monKey, integrations_dir in MONITORS_TO_INTEGRATIONS_DIR.items():
+        package_path, monitor_type = monKey.rsplit(":", 1)
+        for mon in yaml.safe_load((AGENT_ROOT / package_path / "metadata.yaml").read_bytes())["monitors"]:
+            if mon.get("monitorType") == monitor_type:
+                for name, markdown in generate_metric_markdown(mon):
+                    out_path = INTEGRATIONS_REPO / integrations_dir / "docs" / f"{name}.md"
+                    out_path.write_text(markdown)
+                break
+        else:
+            raise ValueError("monitor type %s not found in %s", (monitor_type, package_path))
```

Comment:
Looks like what's there wouldn't work since it's a , instead of a %.

```suggestion
            raise ValueError(f"monitor type {monitor_type} not found in {package_path}")
```

Reply:
```suggestion
            raise ValueError("monitor type %s not found in %s" % (monitor_type, package_path))
```